Sources/iOSClientExposure/EPG/Parameters/FilteredDates.swift

```diff
@@ -58,8 +75,37 @@ public struct DateFilter {
     /// End date in unix epoch time (milliseconds)
     internal let endMillis: Int64
     
-    internal init(start: Int64 = 0, end: Int64 = Date().millisecondsSince1970) {
+    /// Start range date
+    internal let startDate: Date
+    
+    /// Start range date 
+    internal let endDate: Date
+    
+    internal init(
+        start: Int64 = 0,
+        end: Int64 = Date().millisecondsSince1970,
+        startDate: Date =  Date(),
+        endDate: Date = Date()
+    ) {
         self.startMillis = start
         self.endMillis = end
+        self.startDate = startDate
+        self.endDate = endDate
+    }
+    
+    public func daysForward(
+        date: Date = Date()
+    ) -> Int {
+        let calendar = Calendar.current
+        let components = calendar.dateComponents([.day], from: date, to: endDate)
```

Comment:
I think it is rounding down to the nearest whole day. Is this the expected behaviour?